src/test/compile-fail/private-in-public.rs

```diff
@@ -105,8 +105,6 @@ mod aliases_pub {
     }
     impl PrivTr for Priv {}
 
-    // This should be OK, if type aliases are substituted
-    pub fn f2(arg: PrivAlias) {} //~ ERROR private type in public interface
```

Comment:
It's tested, the check is just moved from this file to `private-in-public-warn.rs`